이거는 허락하지 마세요!

SonHyeono opened the pull request.

Commit:
Create QRcodScanner

Commit:
QRcodeScanner

간단한 qr코드 스캐너입니다. import com.google.zxing.integration.android.IntentIntegrator라고 임폴트해서 Intentlntegrator라는 클래스를 가져올수 있도록 해서 스캐너를 만들었습니다.

Commit:
qr코드와는 다른 것들 사장님 activity

activity main에서 사장님 화면 이고 로그인 하고 들어갈시에 보이는 화면
setting은 테이블 개수, 아이디 관리, 메뉴판 수정등을 할수있는 곳이고
call은 손님들이 call버튼을 이용해서 테이블에 가져다줄것들, 메뉴들, 기타 다른 것들을 요청해서 그것들을 보일수 있게 한 layout
boss 버튼은 대기자 명단을 확인가능

Commit:
layout에서 적었던 글귀와 동일

kimnamjun reviewed (approved):
20190930

kimnamjun commented:
???

kimnamjun reviewed (changes requested):
Request Changes
Submit feedback that must be addressed before merging.

kimnamjun reviewed (changes requested):
Request Cheanges 2

Commit:
Add files via upload

qr코드 스캔 layout입니다!

Commit:
Add files via upload

일단은 clickme(id : btn_scan_me)버튼을 누르면 카메라로 넘어가고 qr코드를 인식해서 qr코드에서 나오는 데이터를 인식 할수있고 그 데이터를 firebase에 넘겨야하지만 아직 그까지는 못하고 qr코드를 인식해서 인식되면 activity_confirm으로 넘어가게 해놨습니다.
데이터를 firebase로 넘기도록 다시 해보겠습니다!

Commit:
Add files via upload

reservationapp에다가 qr코드 접목시킨것.
아직까지는 qr코드 찍으면 인식은 되지만 그 데이터를 서버에 연동시키는것은 아직 못함. 단지 qr코드 찍고 인식이되면 activity_confirm으로 화면이 넘어가게만 해둠!
또한 AndroidManifest.xml에다가            <uses-permission android:name="android.permission.CAMERA"/>
 //이걸 적어야지 카메라가 작동 
또한 build.gradle(Module:app)에다가           implementation 'com.journeyapps:zxing-android-embedded:3.6.0'  
이걸 적어야지  qr코드를 위한 IntentIntegrator class를 가져올수 있음!!      git에서 zxing을 가져오는 것

Commit:
Add files via upload

activity_confirm은 number_before, number_time 두개 id만 적음
앱 현황 : qr코드 인식해서 그 data를 confirmactivity에서 확인할수가 있고 또한 finish를 이용해서 activity가 쌓이지 않게 하였음, qr코드 인식후에 다른 어떠한 짓을해도 confirmactivity에 아까 인식한 data가 text하 될수있게 putExtra를 잘해놓음!!
이때 lateinit이라는 개념을 이용했고 또한 변수를 putExtra할때 받을때는 intent.extras?.get("").toString() 를 해야지 받을수 있고
각 activity를 왔다갔다 할때마다 그전 activity는 없애는 대신 "Result" 와 "ResultBack"을 이용해서 data를 손실없이 앱을 구성함.

kimnamjun reviewed (approved):
20191016

kimnamjun commented:
20191017

kimnamjun committed:
Merge branch 'master' into master
